src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -1267,28 +1267,33 @@ public List<AltinnCoreFile> GetImplementationFiles(string org, string service)
                 coreFiles.Add(corefile);
             }
 
-            string[] modelFiles = Directory.GetFiles(_settings.GetModelPath(org, service, AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext)));
-            foreach (string file in modelFiles)
+            string[] modelFiles = null;
+
+            if (Directory.Exists(_settings.GetModelPath(org, service, AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext))))
             {
-                var corefile = new AltinnCoreFile
+                modelFiles = Directory.GetFiles(_settings.GetModelPath(org, service, AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext)));
+                foreach (string file in modelFiles)
                 {
-                    FilePath = file,
-                    FileName = System.IO.Path.GetFileName(file),
-                    LastChanged = File.GetLastWriteTime(file),
-                };
+                    var corefile = new AltinnCoreFile
```

Comment:
Don't use var

Reply:
I see that is was there all along, but please change it when you fix the stylecop errors